### PR TITLE
solution was added

### PR DESCRIPTION
**Pull request by IrynaDim**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+language: java
+jdk:
+  - openjdk11
+script:
+  - mvn clean install
+cache:
+  directories:
+    - $HOME/.m2
```
Comment (AnaLanda) [inline .travis.yml:8]: You should add an empty line at the end of this file)

File: pom.xml [hunk 2]
```diff
@@ -7,6 +7,98 @@
...
+                    <configuration>
+                        <source>${jdk.version}</source>
+                        <target>${jdk.version}</target>
+                        <encoding>${project.build.sourceEncoding}</encoding>
+                    </configuration>
+                </plugin>
+            </plugins>
+        </pluginManagement>
+    </build>
 </project>
```
Comment (AnaLanda) [inline pom.xml:104]: Same here)

File: src/main/java/spring/intro/dao/impl/UserDaoImpl.java [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+            transaction = session.beginTransaction();
+            session.save(user);
+            transaction.commit();
+            logger.info("User was created" + user);
+        } catch (Exception e) {
+            if (transaction != null) {
+                transaction.rollback();
+            }
+            throw new DataProcessingException(
+                    "Cant add user with email " + user.getEmail()
```
Comment (AnaLanda) [inline src/main/java/spring/intro/dao/impl/UserDaoImpl.java:39]:
```suggestion
                    "Can't add user with the email " + user.getEmail()
```

**Review by AnaLanda — COMMENTED** [review]
Looks good 😃

File: pom.xml [hunk 4]
```diff
@@ -7,6 +7,98 @@
...
+                    <configLocation>${maven.checkstyle.plugin.configLocation}</configLocation>
+                    <encoding>UTF-8</encoding>
+                    <consoleOutput>true</consoleOutput>
+                    <failsOnError>true</failsOnError>
+                    <linkXRef>false</linkXRef>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-war-plugin</artifactId>
```
Comment (Sofasmile) [inline pom.xml:82]: do we need this plugin?

Comment (Sofasmile) [reply pom.xml:82]: I told about maven-war-plugin

File: pom.xml [hunk 5]
```diff
@@ -94,5 +86,9 @@
                 </configuration>
             </plugin>
         </plugins>
+        <pluginManagement>
+            <plugins>
+            </plugins>
+        </pluginManagement>
```
Comment (Sofasmile) [inline pom.xml:92]: це також варто видалити: 89-92

**Review by Sofasmile — APPROVED** [review]

**Pull request merged**